.rubocop_todo.yml

```diff
@@ -238,6 +241,20 @@ RSpec/ReceiveNever:
   Exclude:
     - 'spec/beaker/hypervisor/vagrant_spec.rb'
 
+# Offense count: 7
+# Configuration parameters: Include, CustomTransform, IgnoreMethods, IgnoreMetadata.
+# Include: **/*_spec.rb
+RSpec/SpecFilePathFormat:
+  Exclude:
+    - '**/spec/routing/**/*'
```

Comment:
Where does this come from?

Reply:
so this is odd. the directory doesn't exist on my local filesystem. But if I remove it from the todo file and regenerate it, rubocop adds it back :thinking: